main.yml

```diff
@@ -0,0 +1,32 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
```

Comment:
To make it easy to fine tune this file we should probably have it trigger on additional branches (or all?).